xpdacq/glbl.py

```diff
@@ -1,9 +1,11 @@
 import os
 import socket
 import time
-from unittest.mock import MagicMock
+import yaml
```

Comment:
Do we need this import? Isn't the point of the YamlClass so we don't have to deal with yaml at this level?